### PR TITLE
[Auto Update Docker] auto_update_docker_staging_branch_1

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [CIAC-9308](https://jira-dc.paloaltonetworks.com/browse/CIAC-9308)

## Description
Staging branch name: `auto_update_docker_staging_branch_1`

Updated docker images:
>demisto/python3

### CODE REVIEW

File: Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ commonfields:
 name: ArcusTeam
 display: ArcusTeam
 category: Data Enrichment & Threat Intelligence
-description: 'The ArcusTeam API allows the user to inspect connected devices'' attack surface. By feeding device identifiers and the software it runs: DeviceTotal will return a map of the device’s attack surface. DeviceTotal was built from the ground up in order to provide complete visibility into connected devices and mitigate 3rd party risk. DeviceTotal can continuously identify & predict such that the connected device security posture is being assessed, prioritized and mitigated effectively.'
+description: "The ArcusTeam API allows the user to inspect connected devices' attack surface. By feeding device identifiers and the software it runs: DeviceTotal will return a map of the device\u2019s attack surface. DeviceTotal was built from the ground up in order to provide complete visibility into connected devices and mitigate 3rd party risk. DeviceTotal can continuously identify & predict such that the connected device security posture is being assessed, prioritized and mitigated effectively."
```
Comment (JudahSchwartz) [inline Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml:7]: this is weird, @anas-yousef device\u2019s

**Review by JudahSchwartz — COMMENTED** [review]
a couple issues with descriptions, search for 
\u2

**Comment by samuelFain** [comment]
> a couple issues with descriptions, search for \u2

@JudahSchwartz, Its `format`, reverted back these changes.

**Comment by JudahSchwartz** [comment]
@samuelFain note @dorschw mentioned these changes should in fact be done

File: Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml [hunk 2]
```diff
@@ -814,7 +814,7 @@ script:
       required: true
       description: Case Comment
     - name: riskAcceptDate
-      description: Risk Accept Date in ‘yyyy-MM-dd’ format. (applicable only in case of closing a case as Risk Managed)
+      description: "Risk Accept Date in \u2018yyyy-MM-dd\u2019 format. (applicable only in case of closing a case as Risk Managed)"
```
[the remark this answers is not in the document]
Comment (samuelFain) [reply Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml]: Fixed

File: Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml [hunk 3]
```diff
@@ -78,22 +78,22 @@ script:
...
       description: ' Page number'
-      defaultValue: "1"
+      defaultValue: '1'
     - name: sort_order
       auto: PREDEFINED
       predefined:
       - desc
       - asc
-      description: ' Sorting order (“asc”,”desc”)'
+      description: " Sorting order (\u201Casc\u201D,\u201Ddesc\u201D)"
```
[the remark this answers is not in the document]
Comment (samuelFain) [reply Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml]: Fixed

File: Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml (hunk 1 above)
Comment (samuelFain) [reply Packs/ArcusTeam/Integrations/ArcusTeam/ArcusTeam.yml]: Fixed

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**